README-template.md

```diff
@@ -908,6 +902,22 @@ When casting to Timestamp TIME have the same TimeZone issues as DATETIME
        </ul>
    </td>
   </tr>
+  <tr valign="top" id="datatype:map">
+   <td><strong><code>ARRAY&lt;STRUCT&lt;key,value&gt;&gt;</code></strong>
+   </td>
+   <td><strong><code>MapType</code></strong>
+   </td>
+   <td>BigQuery has no MAP type, so like other conversions like Apache Avro and BigQuery Load jobs, the connector converts a Spark Map to a REPEATED STRUCT&lt;key,value&gt;.
```

Comment:
nit: `BigQuery has no MAP type, therefore similar to other conversions like ...`